substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ConfigurationParser.java

```diff
@@ -71,4 +71,4 @@ protected static boolean asBoolean(Object value, String propertyName) {
         }
         throw new JSONParserException("Invalid boolean value '" + value + "' for element '" + propertyName + "'");
     }
-}
+}
```

Comment:
Unnecessary change (and also a checkstyle violation)

Reply:
reverted